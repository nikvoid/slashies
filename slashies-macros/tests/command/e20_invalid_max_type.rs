use slashies_macros::Command;

/// An invalid command
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand {
    /// Field with invalid max type
    #[max = "abc"]
    bad_field: String,

    /// Ok field
    #[max = 0]
    good_field: i64,
}

fn main() {}
