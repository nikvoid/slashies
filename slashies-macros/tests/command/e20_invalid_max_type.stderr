error: Only integer and floating point number "max" values are supported
 --> tests/command/e20_invalid_max_type.rs:8:7
  |
8 |     #[max = "abc"]
  |       ^^^^^^^^^^^
